src/r2mm/mods/ModListSort.ts

```diff
@@ -62,6 +62,8 @@ export default class ModListSort {
                 return a.getAuthorName().localeCompare(b.getAuthorName());
             case SortNaming.CUSTOM:
                 return 0;
+            case SortNaming.LAST_UPDATED:
+                return b.getDateUpdated().toISOString().localeCompare(a.getDateUpdated().toISOString());
```

Comment:
Any specific reason we're doing a string comparison instead of comparing dates directly?

Reply:
I don't remember off the top of my head, but I don't think it was working when I tried to compare dates directly. Not sure why that wouldn't work though, so maybe it wasn't working because I was doing something else wrong. I'll look at it later today and see if I can figure out why I did that.

And as for the commit history, I just had a series of consecutive brain farts lol

Reply:
Oh man this is an update i've been waiting on forever! Crazy to see someone forked it out and tested it :)